Lib/asyncio/proactor_events.py

```diff
@@ -22,6 +23,24 @@
 from .log import logger
 
 
+def _set_socket_extra(transport, sock):
+    transport._extra['socket'] = sock
```

Comment:
Please wrap into the new TransportSocket

Reply:
Done